packages/frameworks/src/frameworks.ts

```diff
@@ -1785,10 +1785,13 @@ export const frameworks = [
     website: 'https://www.sanity.io',
     envPrefix: 'SANITY_STUDIO_',
     detectors: {
-      every: [
+      some: [
         {
           path: 'sanity.json',
         },
+        {
+          path: 'sanity.config.js',
+        },
```

Comment:
```suggestion
        {
          path: 'sanity.config.js',
        },
        {
          path: 'sanity.config.ts',
        },
```

The linked document says `.ts` is supported as well so lets add that

Reply:
I wonder if we should include `.tsx` as well. There are entry points in the config API that assumes React components, so I wouldn't be surprised if people try to stick them into this file.

Reply:
Especially since [we say](https://beta.sanity.io/docs/platform/studio/config) that "All these are valid file suffixes for the studio configuration file: .js , .jsx , .ts , .tsx."

Reply:
On it!